model/simulationtests/autosize_hvac.rb

```diff
@@ -396,18 +396,29 @@ def make_tes_coil(model)
 chw_loop.addSupplyBranchForComponent(chw_storage)
 storage_loop.addDemandBranchForComponent(chw_storage)
 
+# WaterSource
 # TODO: I CANNOT GET autosizeReferenceCapacity to work in E+: see https://github.com/NREL/EnergyPlus/issues/8948
 # Yet it is still reported... so whatever
 plhp_clg = OpenStudio::Model::HeatPumpPlantLoopEIRCooling.new(model)
 plhp_clg.setReferenceCapacity(400000)
 # plhp_clg.autosizeReferenceCapacity
 plhp_clg.autosizeSourceSideReferenceFlowRate
 plhp_clg.autosizeLoadSideReferenceFlowRate
+# plhp_clg.autosizeHeatRecoveryReferenceFlowRate
```

Comment:
How do we disable calling `autosizedHeatRecoveryReferenceFlowRate` for the WaterSource version of these objects?